src/ui/public/persisted_state/persisted_state.js

```diff
@@ -86,6 +86,12 @@ export class PersistedState {
     return this._set(params.key, params.value, true);
   }
 
+  clearAllKeys() {
+    Object.getOwnPropertyNames(this._changedState).forEach(key => {
+      this.set(key, null);
```

Comment:
Maybe we should use `setSilent` here and just trigger one `this.emit('set')` after the loop, so we don't emit a set event for each key.